app/views/search/_sort.html.erb

```diff
@@ -1,11 +1,11 @@
 <div class="dgu-sort">
-  <div class="dgu-sort__apply visuallyhidden">
+  <div class="dgu-sort__apply govuk-visually-hidden">
     <input type="submit" value=<%= t('.accessibility.submit_button') %>/>
   </div>
 
   <div class="dgu-sort__sort-type">
     <label>
-      <span class="visuallyhidden"> <%= t('.accessibility.sort_type_label') %></span>
+      <span class="govuk-visually-hidden"> <%= t('.accessibility.sort_type_label') %></span>
```

Comment:
This PR deployment isn't working, and I don't have context of the UI, why is this label text being visually hidden?

Reply:
It's an accessible label that's sitting next to a search filter. You can see it on this page https://data.gov.uk/search?q= next to the filter dropdown with the selection "best match". I will however make the label change you noted, good spot.

Reply:
I'm still interested to know why the label is hidden. If you look at https://www.gov.uk/search/all?keywords=minister&order=relevance there's a similar 'sort by' option (beneath the result count) and the label is shown.

Reply:
I'm guessing it was just a design choice 😬 I'll re-work it.